.markdownlint.yaml

```diff
@@ -0,0 +1,2 @@
+---
+MD041: false
```

Comment:
What does this do?

Reply:
Disables this rule: [MD041 - First line in file should be a top level heading](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md041). 

We use `##` are our top-level headings with the intention that the content of our tickets is not an H1 on the page. I would assume that is the ticket title.